internal/netsync/manager.go

```diff
@@ -621,10 +645,10 @@ func (m *SyncManager) handleTxMsg(tmsg *txMsg) {
 	// interoperability.
 	txHash := tmsg.tx.Hash()
 
-	// Ignore transactions that we have already rejected.  Do not
-	// send a reject message here because if the transaction was already
-	// rejected, the transaction was unsolicited.
-	if _, exists := m.rejectedTxns[*txHash]; exists {
+	// Ignore transactions that have already been rejected.  Do not send a
+	// reject message here because the transaction was unsolicited if it was
```

Comment:
This is now rebased on top of code which deprecates and stops sending reject messages, so this blurb should be removed.

Reply:
Updated.